package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@toyota-research-institute/lakefront",
   "description": "React UI Components",
-  "version": "0.82.3",
+  "version": "0.82.5",
```

Comment:
Bump version to at least 0.83.0.
Btw: If you want the version to be bumped for you, you can include "feature" in at least one of your PRs (https://github.com/ToyotaResearchInstitute/lakefront/blob/main/CONTRIBUTING.md#versioning)